blueprints/blog-post.yaml

```diff
@@ -0,0 +1,22 @@
+title: Blog Post
+'@extends': default
+
+form:
+  fields:
+    tabs:
+      fields:
+        buttons:
+          type: tab
+          title: Blog Post
+          fields:
+            header.image:

```

Comment:
I'd like this to contain an explanation that this image will be displayed next to the post title/summary in the list of blog posts.

Reply:
I dont think I can add a description at the title level. And I didnt want to crowd the description for the image. I could add a hover help tag for it

Reply:
Okay, let's do the help tag.